README.md

```diff
@@ -10,8 +10,10 @@ You can reach the maintainers of this project at:
 
 - Slack: [#kubeadm](http://kubernetes.slackarchive.io/kubeadm/) and [#sig-cluster-lifecycle](http://kubernetes.slackarchive.io/sig-cluster-lifecycle/)
 - Mailing List: https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle
-- Weekly Zoom Meetings: https://zoom.us/j/166836624 every Tuesday from 17:00 to 18:00 UTC+0
+- Weekly [SIG-Cluster-Lifecycle Meetings](https://zoom.us/j/166836624) every Tuesday from 17:00 to 18:00 UTC+0
   - Meeting Notes: https://docs.google.com/document/d/1deJYPIF4LmhGjDVaqrswErIrV7mtwJgovtLnPCDxP7U
+- Weekly [kubeadm office hours](https://zoom.us/j/166836624) every Wednesday from 17:00 to 18:00 UTC+0
+  - Meeting Notes: https://docs.google.com/document/d/130_kiXjG7graFNSnIAgtMS1G8zPDwpkshgfRYS0nggo
 
 ## Roadmap for v1.6
```

Comment:
This section is outdated as well :)

Reply:
There are other PRs fixing this.

Reply:
Shouldn't we update the community docs and link this one there?  The community repo is the canonical location. 
/cc @roberthbailey

Reply:
https://github.com/kubernetes/community/blob/master/sigs.yaml#L377 - needs TLC.

Reply:
How is the readme in this repo synced when the community docs get updated?

Reply:
I prefer to remove the content of this section and just link to https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle#contact (maybe there is something needs to be modified in that linked section). @kargakis WDYT?

Reply:
@xiangpengzhao I am asking about a different thing - there are two other in-flight PRs fixing this (see https://github.com/kubernetes/kubeadm/pull/667 and https://github.com/kubernetes/kubeadm/pull/602)

Reply:
Sorry, I thought you were talking about the roadmap section specifically. Yeah, I guess I can update this to link back to that doc.

Reply:
updated

Reply:
Sorry for not commenting clearly :)